serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -1 +1,369 @@
+
+.Pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-knative-eventing</artifactId>
+</dependency>
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-kubernetes</artifactId>
+</dependency>
+----
+
+.Gradle
+[source,shell,subs="attributes+"]
+----
+quarkus-kubernetes 'io.quarkus:quarkus-kubernetes:{quarkus_version}'
+quarkus-kubernetes 'org.kie.kogito:kogito-addons-quarkus-knative-eventing:{kogito_version}'
+----
+
+Set the following system properties so the files can be generated:
+
+.System Properties
+[source,shell,subs="attributes+"]
+----
+quarkus.kubernetes.deployment-target=knative
+quarkus.knative.name=greeting-quarkus-kubectl
+----
+
+
+Then, build your application as usual, the following files can be found at `target/kubernetes` directory:
+
+.Knative generated files
+* knative.json
+* knative.yml
+
+
+[source,yaml]
+----
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  annotations:
+    app.quarkus.io/commit-id: 06c3fe8e2dfc42e2211cbcc41224f5a3d6bd1f26
+    app.quarkus.io/build-timestamp: 2022-06-23 - 23:53:38 +0000
+  labels:
+    app.kubernetes.io/name: greeting-quarkus-kubectl
+  name: greeting-quarkus-kubectl
+spec:
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: greeting-quarkus-kubectl
+    spec:
+      containerConcurrency: 0
+      containers:
+        - image: dev.local/kogito/serverless-workflow-greeting-quarkus:1.0-native
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /q/health/live
+              port: null
+              scheme: HTTP
+            initialDelaySeconds: 0
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 10
+          name: greeting-quarkus-kubectl
+          ports:
+            - containerPort: 8080
+              name: http1
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /q/health/ready
+              port: null
+              scheme: HTTP
+            initialDelaySeconds: 0
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 10
+----
+
+NOTE: There is a small bug on the 'Decorate API' where some system properties are not take effect, for more information please take a look at this link:{https://github.com/quarkusio/quarkus/issues/23832}[Quarkus issue].
+After the file generation, you'll might be required to add the `imagePullPolicy` manually before deploying it.
+
+
+Deploying using kubectl:
+[source,shell]
+----
+kubectl apply -f target/kubernetes/knative.yml
+----
+
+Check if the service was deployed correctly:
+[source,shell]
+----
+kubectl get services.serving.knative.dev greeting-quarkus-kubectl
```

Comment:
alternatively, `kn services get...` I think it's easier this way since we have kn installed.